Dockerfile

```diff
@@ -17,4 +17,8 @@ RUN mamba env update -n base --file pytorch.yml \
         torchvision \
         "pytorch-cuda=${CUDA_VER%.*}.*" \
     && conda clean -afy \
-    && rm pytorch.yml
\ No newline at end of file
+    && rm pytorch.yml
+
+
+FROM base as examples
+COPY . .
```

Comment:
IIUC the downside of this `COPY` is that we would want to rebuild this image as notebooks change (which is more than we currently do). I'm personally fine with this. 

An alternative, again IIUC, would be to make this a `git clone ...` where we pull whatever the current status of the repo is. @ntabris maybe you have thoughts?

Reply:
I was thinking you could just have something like

```
FROM coiled/gpu:latest as base
COPY <stuff you want to copy>
```

Reply:
Ah, so a separate `Dockerfile`?

Reply:
I'm curious if there's a benefit to having multiple dockerfiles. Just separation of concerns, or maybe something more than that?

Reply:
I don't think it matters much, that's just what I'd do. Maybe I'm not following what you were asking originally.

Reply:
fwiw, rapids does this: https://github.com/rapidsai/docker/blob/branch-23.10/Dockerfile